src/axom/inlet/examples/nested_structs.cpp

```diff
@@ -282,6 +289,7 @@ dimensions: 2
           center: [4, 5, 6]
         - slice:
             x: 10
+            origin: [40, 50, 60]
```

Comment:
Side question: @estebanpauli -- how do `x` and `origin` combine to create a slicing plane?

Reply:
By default, if you just specify "x: 10", you get the "x=10" plane, with the origin at (10, 0, 0) and an up direction of (0, 1, 0). Setting the origin allows you to move the origin. In the example above, an error would be reported because the x component doesn't match the specified x value.

For full control, you can specify "origin" (point), "normal" (vector), and "up" (vector) to define your plane. The "x", "y", and "z" versions just serve as convenient shortcuts by giving you default values for the unspecified parts of the origin as well as the normal and up vectors.